text/0297-deprecate-ember-logger.md

```diff
@@ -170,15 +161,24 @@ a shim for users.
 
 ## Unresolved questions
 
-How do we deal with `Logger.debug` in the codemod? Do we provide separate 
-options for those who might use Node versions earlier than 9 and those who 
-are confident they will only use Node version 9 or later? Do we have the 
-codemod inject a polyfill for console.debug that calls console.log? Do we
-provide one separately for the user to apply? Or does that become a fastboot 
-concern, since that's the primary driver for running ember projects in node.
-
-What do we do about the eslint `no-console` flag? Some developers are 
-using `Ember.Logger` right now to work around it. Others are providing 
-their own logging service. If we encourage the use of a Babel plugin 
-to strip console calls, should we remove `no-console` from the default 
-flags that `ember-cli` ships?
+How do we deal with `Logger.debug` in the codemod? 
```

Comment:
A suggestion: Try making the question a sub-header and not use bullet points for all of the answer.